src/GovUk.Education.ExploreEducationStatistics.Admin.Tests/Services/ReleaseDataFileServiceTest.cs

```diff
@@ -995,6 +996,211 @@ await using (var contentDbContext = InMemoryApplicationDbContext(contentDbContex
+                {
+                    Type = FileType.Data,
+                    SubjectId = Guid.NewGuid(),
+                },
+            };
+            var releaseDataFile2 = new ReleaseFile
+            {
+                Release = release,
+                Order = 3,
+                File = new File
+                {
+                    Type = FileType.Data,
+                    SubjectId = Guid.NewGuid(),
+                },
+            };
+            var releaseDataFile3 = new ReleaseFile
+            {
+                Release = release,
+                Order = 1,
+                File = new File
+                {
+                    Type = FileType.Data,
+                    SubjectId = Guid.NewGuid(),
+                },
+            };
+            var releaseDataFile4 = new ReleaseFile
+            {
+                Release = release,
+                Order = 2,
+                File = new File
+                {
+                    Type = FileType.Data,
+                    SubjectId = Guid.NewGuid(),
+                },
+            };
+            var releaseDataFile5 = new ReleaseFile
+            {
+                Release = release,
+                Order = 0,
+                File = new File
+                {
+                    Type = FileType.Data,
+                    SubjectId = Guid.NewGuid(),
+                },
+            };
+
+            var releaseMetaFile1 = new ReleaseFile
+            {
+                Release = releaseDataFile1.Release,
+                File = new File
+                {
+                    Type = FileType.Metadata,
+                    SubjectId = releaseDataFile1.File.SubjectId
+                }
+            };
+            var releaseMetaFile2 = new ReleaseFile
+            {
+                Release = releaseDataFile2.Release,
+                File = new File
+                {
+                    Type = FileType.Metadata,
+                    SubjectId = releaseDataFile2.File.SubjectId
+                }
+            };
+            var releaseMetaFile3 = new ReleaseFile
+            {
+                Release = releaseDataFile3.Release,
+                File = new File
+                {
+                    Type = FileType.Metadata,
+                    SubjectId = releaseDataFile3.File.SubjectId
+                }
+            };
+            var releaseMetaFile4 = new ReleaseFile
+            {
+                Release = releaseDataFile4.Release,
+                File = new File
+                {
+                    Type = FileType.Metadata,
+                    SubjectId = releaseDataFile4.File.SubjectId
+                }
+            };
+            var releaseMetaFile5 = new ReleaseFile
+            {
+                Release = releaseDataFile5.Release,
+                File = new File
+                {
+                    Type = FileType.Metadata,
+                    SubjectId = releaseDataFile5.File.SubjectId
+                }
+            };
```

Comment:
I don't think you need to add SubjectId's here anymore as to build the view models, the meta file is now found via the `DataImport`